scripts/install.js

```diff
@@ -1,4 +1,10 @@
+const caniuseYarn = require('@danielbayerlein/caniuse-yarn');
 const executeAll = require('./helper/executeAll');
 const projects = require('./helper/projects');
 
-executeAll('npm', ['install'], projects);
+if (caniuseYarn()) {
+  executeAll('yarn', ['install'], projects);
```

Comment:
i know we could also use a ternary, but it think in this case a if - else is more readable ;)